elibrary_parser/Parsers.py

```diff
@@ -188,7 +188,9 @@ def save_publications(self):
         save_path = self.data_path / "processed" / self.author_id
         save_path.mkdir(exist_ok=True)
 
-        csv_path = save_path / "publications.csv"
+        authors_ids_csv = str(self.author_id) + "_publications.csv"
```

Comment:
Можно 
```suggestion
        authors_ids_csv = f"{self.author_id)}_publications.csv"
```

Это чуть быстрее выполняется и читается, а суть такая же :)